samples/kubernetes/end2end/docs/03-wls-operator.md

```diff
@@ -3,12 +3,18 @@ We depend on the [WLS Kubernetes Operator](https://github.com/oracle/weblogic-ku
 
 In this example, we provide the steps to install the 2.1 release of the WLS Kubernetes Operator.
 
+### Pulling the Images
+We need two images, one is the WLS Kubernetes Operator image from [Docker Hub](https://hub.docker.com) and the other is the WebLogic image from [Oracle Container Registry](https://container-registry.oracle.com).  
+Before pulling the image, you need to:
+- Aquire an user account to the image site.
```

Comment:
If you do not already have Oracle Single Sign-On credentials, navigate to https://container-registry.oracle.com and create your SSO credentials by clicking the Sign In link to create them, at the top of the page.

Reply:
Could you replace 
"Before pulling the image, you need to:
- Aquire an user account to the image site."
with
"If you do not already have Oracle Single Sign-On credentials, navigate to https://container-registry.oracle.com and create your SSO credentials by clicking the Sign In link to create them, at the top of the page."